smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/formurl/MemberShapeEncodeFormURLGenerator.kt

```diff
@@ -88,9 +89,14 @@ abstract class MemberShapeEncodeFormURLGenerator(
                     }
                 }
                 is TimestampShape -> {
-                    val format = TimeStampFormat.determineTimestampFormat(nestedMember, nestedMemberTarget, defaultTimestampFormat)
-                    val encodeValue = "TimestampWrapper($nestedMemberTargetName, format: .$format), forKey: ${ClientRuntimeTypes.Serde.Key}(\"${nestedMemberResolvedName}\")"
-                    writer.write("try $containerName.encode($encodeValue)")
+                    val codingKey = "${ClientRuntimeTypes.Serde.Key}(\"${nestedMemberResolvedName}\")"
+                    val code = TimestampEncodeGenerator(
+                        containerName,
+                        nestedMemberTargetName,
+                        codingKey,
+                        TimestampHelpers.getTimestampFormat(nestedMember, nestedMemberTarget, defaultTimestampFormat)
+                    ).generate()
+                    writer.write(code)
```

Comment:
this is fine here but we should prefer using formatter over generating some string and dumping to writer.